cirq/experiments/fidelity_estimation.py

```diff
@@ -22,13 +22,70 @@
 from cirq.sim import final_wavefunction
 
 
-def linear_xeb_fidelity(
+def linear_xeb_fidelity_estimator(hilbert_space_dimension: int,
+                                  probabilities: Sequence[float]) -> float:
+    """Linear XEB fidelity estimator.
+
+    Estimates fidelity from ideal probabilities of observed bitstrings.
+    The variance of this estimator is
+
+        (1 + 2f - f^2) / M
+
+    where f is the fidelity and M the number of observations, equal to
+    len(probabilities). This is better than logarithmic XEB (see below)
+    when fidelity is f < 0.32.
+
+    The estimator is intended for use with xeb_fidelity() below.
+
+    Args:
+        hilbert_space_dimension: Dimension of the Hilbert space on which
+           the channel whose fidelity is being estimated is defined.
+        probabilities: Ideal probabilities of bitstrings observed in
+            experiment.
+    Returns:
+        Estimate of fidelity associated with an experimental realization
+        of a quantum circuit.
+    """
+    return hilbert_space_dimension * np.mean(probabilities) - 1
+
+
+def log_xeb_fidelity_estimator(hilbert_space_dimension: int,
+                               probabilities: Sequence[float]) -> float:
+    """Logarithmic XEB fidelity estimator.
+
+    Estimates fidelity from ideal probabilities of observed bitstrings.
+    The variance of this estimator is
+
+        (pi^2/6 - f^2) / M
+
+    where f is the fidelity and M the number of observations, equal to
+    len(probabilities). This is better than linear XEB (see above) when
+    fidelity is f > 0.32.
+
```

Comment:
Describe in more detail what this value represents, e.g. the average Bayes prediction loss in bits per sample.

Reply:
Done (described relationship of estimator's MSE to its variance which is particularly simple in this case since the estimators are unbiased).